mlflow/utils/cli_args.py

```diff
@@ -25,3 +25,8 @@
                              "a Conda environmen with the necessary dependencies for "
                              "the current project instead of attempting to create a new "
                              "conda environment.")
+
+INSTALL_MLFLOW = click.option("--install-mlflow", is_flag=True, default=False,
+                              help="If specified, mlflow will be installed into the environment "
```

Comment:
We should mention that this is true only if conda is in use. Btw, L25 environmen --> environment

Reply:
thx. yup makes sense.